<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="5" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default Suite" duration-ms="3867" started-at="2022-01-25T16:06:42 ICT" finished-at="2022-01-25T16:06:46 ICT">
    <groups>
    </groups>
    <test name="AllTest" duration-ms="3867" started-at="2022-01-25T16:06:42 ICT" finished-at="2022-01-25T16:06:46 ICT">
      <class name="PesonTest">
        <test-method status="PASS" signature="Setup()[pri:0, instance:PesonTest@49fc609f]" name="Setup" is-config="true" duration-ms="2312" started-at="2022-01-25T16:06:42 ICT" finished-at="2022-01-25T16:06:44 ICT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="PASS" signature="checkVaccined(int)[pri:0, instance:PesonTest@49fc609f]" name="checkVaccined" duration-ms="771" started-at="2022-01-25T16:06:44 ICT" data-provider="data-provider" finished-at="2022-01-25T16:06:45 ICT">
          <params>
            <param index="0">
              <value>
                <![CDATA[18]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkVaccined -->
        <test-method status="FAIL" signature="checkVaccined(int)[pri:0, instance:PesonTest@49fc609f]" name="checkVaccined" duration-ms="11" started-at="2022-01-25T16:06:45 ICT" data-provider="data-provider" finished-at="2022-01-25T16:06:45 ICT">
          <params>
            <param index="0">
              <value>
                <![CDATA[21]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [18] but found [21]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [18] but found [21]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1240)
at org.testng.Assert.assertEquals(Assert.java:1274)
at PesonTest.checkVaccined(PesonTest.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkVaccined -->
        <test-method status="FAIL" signature="checkVaccined(int)[pri:0, instance:PesonTest@49fc609f]" name="checkVaccined" duration-ms="0" started-at="2022-01-25T16:06:45 ICT" data-provider="data-provider" finished-at="2022-01-25T16:06:45 ICT">
          <params>
            <param index="0">
              <value>
                <![CDATA[51]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [18] but found [51]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [18] but found [51]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1240)
at org.testng.Assert.assertEquals(Assert.java:1274)
at PesonTest.checkVaccined(PesonTest.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkVaccined -->
        <test-method status="PASS" signature="testPersonVaccin()[pri:0, instance:PesonTest@49fc609f]" name="testPersonVaccin" duration-ms="0" started-at="2022-01-25T16:06:45 ICT" description="Ban Tran Lam da duoc chich vaccin" finished-at="2022-01-25T16:06:45 ICT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPersonVaccin -->
        <test-method status="PASS" signature="testPesonVaccin2()[pri:0, instance:PesonTest@49fc609f]" name="testPesonVaccin2" duration-ms="0" started-at="2022-01-25T16:06:45 ICT" description="Ban Tran Long da duoc chich vaccin" finished-at="2022-01-25T16:06:45 ICT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPesonVaccin2 -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:PesonTest@49fc609f]" name="TearDown" is-config="true" duration-ms="612" started-at="2022-01-25T16:06:45 ICT" finished-at="2022-01-25T16:06:46 ICT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- PesonTest -->
    </test> <!-- AllTest -->
  </suite> <!-- Default Suite -->
</testng-results>
